zone/gm_commands/parcels.cpp

```diff
@@ -184,30 +203,34 @@ void command_parcels(Client *c, const Seperator *sep)
 
 			Parcel_Struct ps{};
 			ps.item_slot = parcel_out.slot_id;
-			strn0cpy(ps.send_to, parcel_out.to_name.c_str(), sizeof(ps.send_to));
+			strn0cpy(ps.send_to,parcel_out.to_name.c_str(),sizeof(ps.send_to));
```

Comment:
Spaces removed?

Reply:
I have been having regular fights with the auto formatting.  A royal PITA in fact.  :-)  When I autoformat, it puts everything on separate lines.  Is the standard separate lines or a single?  I was working under the impression that if it was 80 chars or less, then the auto formatter should leave it on a single line?  @Kinglykrab

Reply:
Updated